packages/plugin-interaction-breadcrumbs/interaction-breadcrumbs.js

```diff
@@ -21,8 +21,7 @@ module.exports = (win = window) => ({
   }
 })
 
-const trimStart = /^\s+/
-const trimEnd = /(^|[^\s])\s+$/
+const trim = /^\s*([^\s].{0,139}[^\s])?\s*/s
```

Comment:
@lemnik looks like we've got an issue with this regex - perhaps the `s` at the end? it was modified slightly from the suggestion as it didn't remove the trailing whitespace.

Reply:
Yeah, that's certainly what the error indicated, although the `/s` flag is the [dotAll](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/dotAll) flag. Maybe it's a version compatibility thing and we need to use `[.\n]` instead of `.` (assuming `dotAll` is not available on the target browser)?

Reply:
I replaced `.` with `[.\n]` and tests failed locally.
 For example:
<img width="593" alt="Screenshot 2024-03-11 at 16 32 24" src="https://github.com/bugsnag/bugsnag-js/assets/119674359/37a39c3a-6835-48f0-a0c6-9cd98a553b2d">

Reply:
It worked! Thanks!